### DISCUSSION TITLE
Stand Up Tuesday June 6th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#102] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- refined most attended certifications graph
- tried to develop another algorithm based on word2vec that compared certifications by their name

**What are your priorities for the next 24 hours?**:
- work on rest of graphs with @Jacobgtz024 

**Blockers**:
- will start working at 5 pm

**Shoutouts**:
- @username for x

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on @CristianCazares  on matched_certification graph
- Finished Performance Testign suite for backend
- Almost done with Adan's document. 

**What are your priorities for the next 24 hours?**:
-  Finish adans activity
-  Work on more graphs

**Blockers**:
-  None

**Shoutouts**:
- @CarlosCRG19 for getting a new pair of jeans

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Started working on the "new" enpoints

**What are your priorities for the next 24 hours?**:
- Work with @CarlosCRG19 to finish them

**Blockers**:
- None

**Shoutouts**:
- @todos por guapos

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Completed the Dashboard Tasks activity
- Finally set up local backen (again) and data analysis service
- Worked with @BrandonMagana on refactoring and creating the metched_certification graph on back and data analysis service
- Refactor some frontend stuff
- Added the matched_certification graph on front

**What are your priorities for the next 24 hours?**:
- Next graph
- Work in redis doc

**Shoutouts**:
- @BrandonMagana for helping out with my local services
